easybuild/easyconfigs/s/SNP2HLA/SNP2HLA-v1.0.3.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = "Tarball"
+
+name = 'SNP2HLA'
+version = 'v1.0.3'
+
+homepage = 'http://software.broadinstitute.org/mpg/snp2hla/'
+description = ''
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://software.broadinstitute.org/mpg/snp2hla/data/']
+sources = ['SNP2HLA_package_v1.0.3.tar.gz']
+checksums = ['b8ca677860e37f4437f54bd3bd475ade']
+
+#postinstallcmds = ['cd %(installdir)s/source ',
+#	'g++ -O5 -o cranefoot main.cc *.cpp -lm'
+#	]
+
+sanity_check_paths = {
+    'files': ["README.txt"],
```

Comment:
Please test at least for the installed binary. Making sure the README is there is nice, but in this case the installation may silently fail to install the actual software.